src/aiperf/common/config/input_config.py

```diff
@@ -98,6 +98,39 @@ def validate_custom_dataset_file(self) -> Self:
             raise ValueError("Custom dataset type requires --input-file to be provided")
         return self
 
+    @model_validator(mode="after")
+    def validate_rankings_token_options(self) -> Self:
```

Comment:
I think you should validate that the endpoint is a rankings endpoint as well, and fail if the user set any of the rankings options.